src/kv/RocksDBStore.cc

```diff
@@ -1378,7 +1378,7 @@ void RocksDBStore::RocksDBTransactionImpl::set(
   } else {
     string key;
     combine_strings(prefix, k, keylen, &key);
-    put_bat(bat, cf, key, to_set_bl);
+    put_bat(bat, db->default_cf, key, to_set_bl);
```

Comment:
i think it's fine passing a `nullptr` here, as the rockdb uses the nullptr `ColumnFamilyHandle` as a special case. see https://github.com/facebook/rocksdb/blob/722814e357104c3e428b31740b4627239924b973/include/rocksdb/write_batch.h#L78-L82 . before introducing column family support, we were using `Put(const SliceParts& key, const SliceParts& value)`.
after that change, we only use the lower level variant API which accepts `cf` . but it's practically optional.

so as a cleanup, probably you could `s/cf/nullptr/` to make explicit? and note in the commit message that it's a cleanup for better readability?